limitador-server/script/kind-setup.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This script deploys the environment described in
+# limitador-server/kubernetes/README.md in a local cluster using kind
+# (https://github.com/kubernetes-sigs/kind)
+
+SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+CLUSTER_NAME=limitador
+LIMITADOR_NAMESPACE=default
+
+kind delete cluster --name ${CLUSTER_NAME}
+kind create cluster --name ${CLUSTER_NAME}
+
+# Deploy Redis
+kubectl apply -f "${SCRIPT_DIR}"/../kubernetes/redis-service.yaml
+kubectl apply -f "${SCRIPT_DIR}"/../kubernetes/redis-statefulset.yaml
+
+# Deploy Limitador
+kubectl apply -f "${SCRIPT_DIR}"/../kubernetes/limitador-config-configmap.yaml
+kubectl apply -f "${SCRIPT_DIR}"/../kubernetes/limitador-service.yaml
+kubectl apply -f "${SCRIPT_DIR}"/../kubernetes/limitador-deployment.yaml
+
+# Deploy Kuard. In kind we don't use the proxy protocol, so we need to disable
+# the option in the Envoy config
+sed 's/use_proxy_proto: true/use_proxy_proto: false/g' "${SCRIPT_DIR}"/../kubernetes/kuard-envoy-config-configmap.yaml | kubectl apply -f -
+kubectl apply -f "${SCRIPT_DIR}"/../kubernetes/kuard-service.yaml
+kubectl apply -f "${SCRIPT_DIR}"/../kubernetes/kuard-deployment.yaml
```

Comment:
I see you are using on some kubectl commands the `-n ${LIMITADOR_NAMESPACE}` to determine in which namespace resources are deployed, so it should be used on all places, just in case you override `LIMITADOR_NAMESPACE` env var, it still works:
```suggestion
kubectl -n $ apply -f "${SCRIPT_DIR}"/../kubernetes/redis-service.yaml
kubectl -n $ apply -f "${SCRIPT_DIR}"/../kubernetes/redis-statefulset.yaml

# Deploy Limitador
kubectl -n $ apply -f "${SCRIPT_DIR}"/../kubernetes/limitador-config-configmap.yaml
kubectl -n $ apply -f "${SCRIPT_DIR}"/../kubernetes/limitador-service.yaml
kubectl -n $ apply -f "${SCRIPT_DIR}"/../kubernetes/limitador-deployment.yaml

# Deploy Kuard. In kind we don't use the proxy protocol, so we need to disable
# the option in the Envoy config
sed 's/use_proxy_proto: true/use_proxy_proto: false/g' "${SCRIPT_DIR}"/../kubernetes/kuard-envoy-config-configmap.yaml | kubectl -n $ apply -f -
kubectl -n $ apply -f "${SCRIPT_DIR}"/../kubernetes/kuard-service.yaml
kubectl -n $ apply -f "${SCRIPT_DIR}"/../kubernetes/kuard-deployment.yaml
```

Please feel free to test that everything works as expected with that changes

Reply:
You're right. I forgot to add it in a few places.
It's fixed now :+1: